store/view/articles.js

```diff
@@ -0,0 +1,168 @@
+  totalItems: 0,
+  mainpage: [],
+  mainlist: [],
+  all: [],
+  byCategory: {},
+  loading: false,
+  error: false,
+  currArticleCode: '',
+  currentCategoryCode: ''
+})
+
+// Module actions
+export const actions = {
+  getMainpage: composer.compose({
+    before: [
+      compounds.callActionWhen(({ ctx }) => _.isEmpty(ctx.getters.mainpage)),
+      compounds.doSth(({ ctx }) => ctx.commit(loading(true)))
+    ],
+    caller: callers.squashErrorsCall('#getMainpage'),
+    success: [
+      opuses.doSth(({ ctx, result }) => ctx.dispatch('addCodes', { path: 'mainpage', codes: result })),
+      opuses.doSth(({ ctx, result }) => ctx.dispatch('addCodes', { path: 'all', codes: result })),
+      opuses.doSth(({ ctx, result }) => ctx.commit(loading(false)))
+    ]
+  })(async function getMainpageAction ({ state, commit, dispatch }, params) {
+    const data = await knitService.getCollection(this.$axios, 'articles', settings.defaultQsObject)
+
+    // Inject articles and return their codes
+    return dispatch('resources/injectResource', {
+      path: 'articles',
+      data: await dispatch('prepareArticles', data)
+    }, { root: true })
+  }),
+  getCategoryList: composer.compose({
+    before: [
+      compounds.callActionWhen(({ ctx, params }) => ctx.getters.currentCategoryCode !== params.categoryCode),
+      compounds.doSth(({ ctx, params }) => ctx.commit(changeCurrCategory(params.categoryCode))),
+      compounds.callActionWhen(({ ctx }) => _.isEmpty(ctx.getters.byCategory[ctx.getters.currentCategoryCode])),
+      compounds.doSth(({ ctx }) => ctx.commit(loading(true)))
+    ],
+    caller: callers.squashErrorsCall('#getCategoryList'),
+    success: [
+      opuses.doSth(({ ctx, params, result }) => ctx.dispatch('addCodes', { categoryCode: params.categoryCode, codes: result })),
+      opuses.doSth(({ ctx, result }) => ctx.dispatch('addCodes', { path: 'all', codes: result })),
+      opuses.doSth(({ ctx, result }) => ctx.commit(loading(false)))
+    ]
+  })(async function getCategoryListAction ({ state, commit, dispatch }, params) {
+    const data = await knitService.getCollection(this.$axios, 'articles', { ...settings.defaultQsObject, 'category.code': params.categoryCode })
+
+    // Inject articles and return their codes
+    return dispatch('resources/injectResource', {
+      path: 'articles',
+      data: await dispatch('prepareArticles', data)
+    }, { root: true })
+  }),
+  getMainlist: composer.compose({
+    before: [
+      compounds.doSth(({ ctx }) => ctx.commit(changeCurrCategory('wszystkie'))),
+      compounds.callActionWhen(({ ctx }) => _.isEmpty(ctx.getters.mainlist)),
+      compounds.doSth(({ ctx }) => ctx.commit(loading(true)))
+    ],
+    caller: callers.squashErrorsCall('#getMainlist'),
+    success: [
+      opuses.doSth(({ ctx, result }) => ctx.dispatch('addCodes', { path: 'mainlist', codes: result })),
+      opuses.doSth(({ ctx, result }) => ctx.dispatch('addCodes', { path: 'all', codes: result })),
```

Comment:
Either `addCodes` have issue in logger string, which says: adding to 'all' and '${path}`
or you don't need this line

Reply:
Logger has wrong string thanks :+1: